torch/csrc/distributed/rpc/init.cpp

```diff
@@ -373,15 +377,17 @@ If the future completes with an error, an exception is thrown.
       [](const WorkerInfo& dst,
          std::string& pickledPythonUDF,
          std::vector<torch::Tensor>& tensors,
-         const std::shared_ptr<torch::autograd::profiler::RecordFunction>& rf) {
+         const std::shared_ptr<torch::autograd::profiler::RecordFunction>& rf,
+         const std::chrono::milliseconds& rpcTimeout) {
         DCHECK(!PyGILState_Check());
-        return pyRpcPythonUdf(dst, pickledPythonUDF, tensors, rf);
+        return pyRpcPythonUdf(dst, pickledPythonUDF, tensors, rf, rpcTimeout);
       },
       py::call_guard<py::gil_scoped_release>(),
       py::arg("dst"),
       py::arg("pickledPythonUDF"),
       py::arg("tensors"),
-      py::arg("rf") = nullptr);
```

Comment:
This is always called with a RecordFunction (`None` if there is no profiling), so just decided to remove it. This will be going away with https://github.com/pytorch/pytorch/pull/35055 as well